.changeset/angry-boats-wash.md

```diff
@@ -0,0 +1,7 @@
+---
+'@primer/react': patch
+---
+
+Include `border*Color` properties in sx to support named Primer colors
+
+<!-- Changed components: _none_ -->
```

Comment:
In this case I think `_all_` might have been the better option. If you agree and see that happening frequently let me know and I'll add the option to the changesets CLI

Reply:
I had a few PRs that I thought "all" would make more sense but some were a bit vague 🤔 Similar to this case like it does affect almost all components but it doesn't really feel a component change to me maybe more of a system change that can be communicated with a different keyword? Curious to hear what you think!